VKSample/VKSample/Modules/FriendPhotosGalleryModule/PhotosGalleryCollectionViewController.swift

```diff
@@ -20,8 +20,9 @@ final class PhotosGalleryCollectionViewController: UICollectionViewController {
     // MARK: - Private properties.
 
     private let networkService = VKNetworkService()
```

Comment:
```suggestion
    private let networkService = NetworkService()
```
либо 
private let vkNetworkService = VKNetworkService()